lib/ui/cachedblock.js

```diff
@@ -0,0 +1,55 @@
+const Block = require('./block');
+const { blockSize } = require('./variables');
+
+class CachedBlock extends Block {
```

Comment:
I don't think `Block` class is going anywhere especially if we use RE:DOM. It just requires some tweaking.